src/components/FloatingHelper/ClosablePopover/ClosablePopover.driver.ts

```diff
@@ -0,0 +1,16 @@
+import {BaseDriver, DriverFactory} from 'wix-ui-test-utils/driver-factory';
+import {popoverDriverFactory} from 'wix-ui-core/dist/src/components/Popover/Popover.driver';
+
+// TODO: add interface of PopoverDriver (not defined yet in the core)
+export interface ClosablePopoverDriver extends BaseDriver {
```

Comment:
Better to add an interface for the PopoverDriver, but it can be done in a future pr

Reply:
Popover in the core has many dependencies, it a big PR.